dist/tools/codespell/ignored_words.txt

```diff
@@ -186,3 +186,6 @@ shiftIn
 
 # <b>e</b>lements (used in Congure documentation)
 lements
+
+# EMAC (ESP32 EMAC layer) => EMACS
+EMAC
```

Comment:
I think this was added recently to master already.

Note: I think codespell needs the exceptions in lower case. At least previous releases didn't work when the exception was added in upper cases (even when all instances in the code where upper case only).

Reply:
Ok, I have to rebase obviously. Regarding uppercase, codespell had complained before this extension, but not with this extension.